extensions/src/main/java/org/pentaho/platform/web/http/api/resources/UsersWrapper.java

```diff
@@ -0,0 +1,48 @@
+/*!
```

Comment:
The same as above. The current format -
 https://help.pentaho.com/Documentation/7.0/0R0/070/010/0D0/0N0#path__userrolelist_getUsersInRole.html . The example of format we will get if we use UserListWrapper - https://help.pentaho.com/Documentation/7.0/0R0/070/010/0D0/0N0#path__userrolelist_users.html